### PR TITLE
Feature/gdh 293 anchor mobile html

### PR DESCRIPTION
**Pull request by paulAcato**
The anchor-navigation is actually 2 components:

1. The anchor-navigation (sidebar for desktop);
2. The collapses for mobile, which will show the content when on desktop;

### CODE REVIEW

File: components/AnchorCollapse/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@gemeente-denhaag/anchor-collapse",
+  "version": "0.1.0",
+  "description": "An anchor collapse component",
+  "bugs": "https://github.com/nl-design-system/denhaag/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nl-design-system/denhaag.git",
+    "directory": "components/AnchorNavigation"
```
Comment (robbieatacato) [inline components/AnchorCollapse/package.json:9]: I think this should be `AnchorCollapse`

File: components/AnchorCollapse/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
+.denhaag-anchor-collapse {
+  --denhaag-anchor-collapse-box-shadow: inset 0 -1px 0 var(--denhaag-anchor-collapse-border-color);
```
Comment (robbieatacato) [inline components/AnchorCollapse/src/index.scss:2]: I am not a fan of box-shadows used as borders. 
If shadows are needed for aesthetic purpose besides the default styling, I'd use box-shadows.
But I guess this is a preferred thing.

File: components/AnchorCollapse/src/index.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  --denhaag-anchor-collapse-title-display: none;
+  --denhaag-anchor-collapse-summary-display: block;
+  --denhaag-anchor-collapse-summary-pointer-events: auto;
+  --denhaag-anchor-collapse-rotate: -180deg;
```
Comment (robbieatacato) [inline components/AnchorCollapse/src/index.scss:6]: This value can be put in the tokens.

File: components/AnchorCollapse/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  --denhaag-anchor-collapse-box-shadow: inset 0 -1px 0px var(--denhaag-anchor-collapse-border-color),
+    inset 0 1px 0px var(--denhaag-anchor-collapse-border-color);
+}
+
+@media (min-width: 768px) {
+  .denhaag-anchor-collapse {
+    --denhaag-anchor-collapse-border-color: transparent;
+    --denhaag-anchor-collapse-title-display: block;
+
+    // Summary
```
Comment (robbieatacato) [inline components/AnchorCollapse/src/index.scss:21]: comments can be removed

File: components/AnchorCollapse/src/index.scss [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    --denhaag-anchor-collapse-summary-display: none;
+    --denhaag-anchor-collapse-summary-pointer-events: none;
+    --denhaag-anchor-collapse-summary-padding-block: 0;
+    --denhaag-anchor-collapse-summary-padding-inline: 0;
+
+    // Content
```
Comment (robbieatacato) [inline components/AnchorCollapse/src/index.scss:27]: comments can be removed

File: components/AnchorNavigation/src/stories/bem.stories.mdx [hunk 6]
```diff
@@ -29,7 +29,7 @@ import "../index.scss";
   {/* eslint-disable react/no-unknown-property */}
   <Story name="Default">
     <nav class="denhaag-anchor-navigation" aria-labelledby="secondary-navigation">
-      <ul id="secondary-navigation" class="denhaag-anchor-navigation__list">
+      <ol id="secondary-navigation" class="denhaag-anchor-navigation__list">
```
Comment (robbieatacato) [inline components/AnchorNavigation/src/stories/bem.stories.mdx:32]: This is a unnecessary change.

**Review by robbieatacato — CHANGES_REQUESTED** [review]

File: components/AnchorCollapse/package.json (hunk 1 above)
Comment (paulAcato) [reply components/AnchorCollapse/package.json:9]: Sharp! Missed that one. Will update right away.

File: components/AnchorNavigation/src/stories/bem.stories.mdx (hunk 6 above)
Comment (paulAcato) [reply components/AnchorNavigation/src/stories/bem.stories.mdx:32]: I edit this one because it refers to content on the same page which is ordered.

File: components/AnchorCollapse/src/index.scss (hunk 5 above)
Comment (paulAcato) [reply components/AnchorCollapse/src/index.scss:27]: Will do

File: components/AnchorCollapse/src/index.scss (hunk 2 above)
Comment (paulAcato) [reply components/AnchorCollapse/src/index.scss:2]: edited

File: components/AnchorCollapse/src/index.scss (hunk 4 above)
Comment (paulAcato) [reply components/AnchorCollapse/src/index.scss]: removed them

**Review by robbieatacato — APPROVED** [review]
